src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,37 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
+  displayedTodo: Todo | null;
+  changeShowing: (number: number) => void;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  displayedTodo,
+  changeShowing,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [loadingUser, setLoadingUsers] = useState(false);
+
+  useEffect(() => {
+    if (displayedTodo !== null) {
```

Comment:
```suggestion
    if (displayedTodo) {
```

should work same but simplified